acceptance/workspace/jobs/create/server.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "method": "POST",
+    "path": "/api/2.1/jobs/create",
+    "request": {
```

Comment:
This could easily be made a YAML configuration file as well in the future if we'd like to. If the server configurations get really complex.